Offline wiki top n articles

mrober01 opened the pull request:
Added implementation to have top N full articles in workflow when using  that function i added to OfflineWikipediaAPI

SomeOddCodeGuy commented:
Awesome! I really appreciate you adding this in. I won't get a chance to peek at it until tomorrow or Saturday, but just wanted to let you know that I definitely see it and will jump on it as soon as I can.

mrober01 commented:
Great, sounds good

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Chris ***@***.***>
Sent: Thursday, January 9, 2025 5:43:26 PM
To: SomeOddCodeGuy/WilmerAI ***@***.***>
Cc: mrober01 ***@***.***>; Author ***@***.***>
Subject: Re: [SomeOddCodeGuy/WilmerAI] Offline wiki top n articles (PR #29)


Awesome! I really appreciate you adding this in. I won't get a chance to peek at it until tomorrow or Saturday, but just wanted to let you know that I definitely see it and will jump on it as soon as I can.

—
Reply to this email directly, view it on GitHub<https://github.com/SomeOddCodeGuy/WilmerAI/pull/29#issuecomment-2581570160>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADS62OPOPQYW4QPQTACCOCT2J4QT5AVCNFSM6AAAAABU23AIM2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOBRGU3TAMJWGA>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

SomeOddCodeGuy commented:
On the readme, under this section:

https://github.com/SomeOddCodeGuy/WilmerAI/blob/master/README.md#full-text-wikipedia-offline-api-caller-node

Could you add an example json for the OfflineWikiApiTopNFullArticles node type?

mrober01 commented:
Sure, let me do that.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Chris ***@***.***>
Sent: Saturday, January 11, 2025 4:54:15 PM
To: SomeOddCodeGuy/WilmerAI ***@***.***>
Cc: mrober01 ***@***.***>; Author ***@***.***>
Subject: Re: [SomeOddCodeGuy/WilmerAI] Offline wiki top n articles (PR #29)


On the readme, under this section:

https://github.com/SomeOddCodeGuy/WilmerAI/blob/master/README.md#full-text-wikipedia-offline-api-caller-node

Could you add an example json for the OfflineWikiApiTopNFullArticles node type?

—
Reply to this email directly, view it on GitHub<https://github.com/SomeOddCodeGuy/WilmerAI/pull/29#issuecomment-2585500728>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADS62OL35MH5D36TMCF3DHL2KG4LPAVCNFSM6AAAAABU23AIM2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOBVGUYDANZSHA>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

mrober01 commented:
Hi,
I have updated the readme file and created another pull request for that.  Also, I updated the readme from your OfflineWikipediaTextApi repo in the same way going with the pull request I had submitted there too.  I guess when you update master with these, might want to mention the version / commit date of OfflineWikipediaTextApi that is needed for the Wilmar update.

Thanks,
Mike

________________________________
From: Chris ***@***.***>
Sent: Saturday, January 11, 2025 4:54 PM
To: SomeOddCodeGuy/WilmerAI ***@***.***>
Cc: mrober01 ***@***.***>; Author ***@***.***>
Subject: Re: [SomeOddCodeGuy/WilmerAI] Offline wiki top n articles (PR #29)


On the readme, under this section:

https://github.com/SomeOddCodeGuy/WilmerAI/blob/master/README.md#full-text-wikipedia-offline-api-caller-node

Could you add an example json for the OfflineWikiApiTopNFullArticles node type?

—
Reply to this email directly, view it on GitHub<https://github.com/SomeOddCodeGuy/WilmerAI/pull/29#issuecomment-2585500728>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADS62OL35MH5D36TMCF3DHL2KG4LPAVCNFSM6AAAAABU23AIM2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOBVGUYDANZSHA>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
